.gitignore

```diff
@@ -6,3 +6,6 @@ __pycache__
 build
 dist
 pip-log.txt
+.idea
```

Comment:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore :)

Reply:
thanks for calling that out. I didn't know about that!